README.md

```diff
@@ -128,5 +128,11 @@ Open `cpp/demos/ossrf-nmos-api/config/nmos_config.json` and adjust the following
 
     ./scripts/build-inside-container.sh
 
+## GStreamer Plugins
+
+We have also developed GStreamer plugins (nmossender, nmosvideoreceiver, and nmosaudioreceiver) that integrate with NMOS for audio/video sending and receiving. 
+
```

Comment:
We have also developed GStreamer plugins (nmossender, nmosvideoreceiver, and nmosaudioreceiver) **that integrate NMOS registration and control with the sending and receiving of raw ST2110 audio/video streams.**